kinto/config/__init__.py

```diff
@@ -9,7 +9,8 @@ def render_template(template, destination, **kwargs):
     template = os.path.join(HERE, template)
 
     folder = os.path.dirname(destination)
-    os.makedirs(folder)
+    if os.path.exists(folder) == False:
```

Comment:
We usually don't compare with boolean values, but use negation like `if not os.path.exists(folder)`